src/Server/modules/USPlacesManager.js

```diff
@@ -0,0 +1,331 @@
+						'/' +
+						temp.getDate() +
+						'/' +
+						temp.getFullYear();
+					if (d == date) {
+						allUSPlaces[i].currentNumDeaths[key] = number;
+						result = true;
+						break;
+					}
+				}
+			}
+		}
+	}
+	return result;
+}
+
+
+function RemoveUSData(county, state, date, tod) {
+    if(allUSPlaces == undefined){
+        InitializeAllUSPlace();
+    }
+	var result = false;
+	if (tod == 1) {
+		console.log(allUSPlaces.length);
+		for (var i = 0; i < allUSPlaces.length; ++i) {
+			if (county == allUSPlaces[i].county && state == allUSPlaces[i].state) {
+				for (var key in allUSPlaces[i].numConfirmed) {
+					var temp = new Date(key);
+					var d =
+						temp.getMonth() +
+						1 +
+						'/' +
+						temp.getDate() +
+						'/' +
+						temp.getFullYear();
+					if (d == date) {
+						delete allUSPlaces[i].numConfirmed[key];
+						result = true;
+						break;
+					}
+				}
+			}
+		}
+	} else if (tod == 2) {
+		for (var i = 0; i < allUSPlaces.length; ++i) {
+			if (county == allUSPlaces[i].county && state == allUSPlaces[i].state) {
+				for (var key in allUSPlaces[i].numDeaths) {
+					var temp = new Date(key);
+					var d =
+						temp.getMonth() +
+						1 +
+						'/' +
+						temp.getDate() +
+						'/' +
+						temp.getFullYear();
+					if (d == date) {
+						delete allUSPlaces[i].numDeaths[key];
+						result = true;
+						break;
+					}
+				}
+			}
+		}
+	}
+	return result;
+}
+// function SaveRecords()
+// A void function that saves allUSPlace (Maybe modified) into csv.
+function SaveRecords(){
+    if(allUSPlaces == undefined){
+        InitializeAllUSPlace();
+    }
+    writeCSVModule.RecordUSData(allUSPlaces);
+}
+
+function GetTwoPlacesComparison(state1, county1, state2, county2, typeOfData) {
+    if(allUSPlaces == undefined){
+        InitializeAllUSPlace();
+    }
+    var county1Data = null;
+    var county2Data = null;
+
+        for (var i = 0; i < allUSPlaces.length; ++i) {
+            if (allUSPlaces[i].state == state1 && allUSPlaces[i].county == county1) {
+                county1Data = allUSPlaces[i];
+            }
+            if (allUSPlaces[i].state == state2 && allUSPlaces[i].county == county2) {
+                county2Data = allUSPlaces[i];
+            }
+        }
+
+    if (county1Data == null || county2Data == null) {
+        place1Value = (county1Data == null) ? -1 : 0;
+        place2Value = (county2Data == null) ? -1 : 0;
+        return { place1Value: place1Value, place2Value: place2Value }
+    }
+
+    var place1Value, place2Value;
+
+    switch (typeOfData) {
+        case '1': // confirmed
+            place1Value = county1Data.currentNumConfirmed;
+            place2Value = county2Data.currentNumConfirmed;
+            break;
+        case '2': // dead
+            place1Value = county1Data.currentNumDeaths;
+            place2Value = county2Data.currentNumDeaths;
+            break;
+        case '3': // recovered
+            place1Value = county1Data.currentNumRecovered;
```

Comment:
There should not be recovered value. currentNumRecovered does not exist for US data. We can remove case '3'.